lib/recommender_system/irt_recommender.rb

```diff
@@ -0,0 +1,28 @@
+module RecommenderSystem
+  class IrtRecommender < RecommenderSystem::Recommender
+    def get_probability
+      los = self.learning_objects
+      list = Hash.new
+      los.each do |lo|
+        irt_values = lo.irt_values.first
```

Comment:
Co sa tyka irt_value, zatial to tam nie je, ale LO bude mat viacero zaznamov, takze tam doplnim timestamps a budem brat posledny.